configure

```diff
@@ -5736,10 +5739,15 @@ done
 
 if test $have_pocketsphinx = yes -a $have_speech_recognizer = yes -a $enable_speech_recognizer = yes; then
    BUILD_SPEECH_RECOGNIZER="speech_recognizer"
-   POCKETSPHINX_BASE_LIBS="-lpocketsphinx"
+   RECOGNIZER_BASE_LIBS=`pkg-config --libs pocketsphinx sphinxbase`
+   RECOGNIZER_BASE_CFLAGS=`pkg-config --cflags pocketsphinx sphinxbase`
+   RECOGNIZER_ENGINE_CLASS=PocketsphinxSpeechRecognitionEngine.m
```

Comment:
If you would use just the class name here you could use this variable to create an instance in GSSpeechRecognitionEngine with NSClassFromString().

Reply:
One of the things which was thought of, but not implemented in the speech synthesis system is to use bundles instead of linking the class directly.  What I am going to do is the following:

I am going to create a bundles directory in speech_synthesizer.   I am going to move the engine into that and separate out everything into a bundle for pocketsphinx.  Right now this will be the only one, but it will make it easy to add them and, ultimately, select between them.   We could even do speech recognizer bundles in programs separately or have them in their own directory in the system library.

Just some thoughts.   Right now it's proving to be a pain to add this so that the .m file can see it.   So why add something now that I am going to feel compelled to remove later.

Reply:
I'm going to defer this one until later.   The bundle approach should work it's just not loading fast enough.